commit 235c27358c0ca04e35cc971d1150e3bd10d6c0f6
Author: Thomas J. Moore <darktjm@gmail.com>
Date:   Sun Nov 15 19:42:24 2020 -0600

    Don't ask gog to sort games, as that modifies the "preferred" sort order

diff --git a/src/website.cpp b/src/website.cpp
index bc85639..5cdaf16 100644
--- a/src/website.cpp
+++ b/src/website.cpp
@@ -95,7 +95,7 @@ std::vector<gameItem> Website::getGames()
 
     do
     {
-        std::string response = this->getResponse("https://www.gog.com/account/getFilteredProducts?hiddenFlag=" + std::to_string(iHidden) + "&isUpdated=" + std::to_string(iUpdated) + "&mediaType=1&sortBy=title&system=&page=" + std::to_string(i));
+        std::string response = this->getResponse("https://www.gog.com/account/getFilteredProducts?hiddenFlag=" + std::to_string(iHidden) + "&isUpdated=" + std::to_string(iUpdated) + "&mediaType=1&system=&page=" + std::to_string(i));
         std::istringstream json_stream(response);
 
         try {
@@ -242,10 +242,7 @@ std::vector<gameItem> Website::getGames()
     } while (!bAllPagesParsed);
     std::cerr << std::endl;
 
-    if (Globals::globalConfig.bIncludeHiddenProducts)
-    {
-        std::sort(games.begin(), games.end(), [](const gameItem& i, const gameItem& j) -> bool { return i.name < j.name; });
-    }
+    std::sort(games.begin(), games.end(), [](const gameItem& i, const gameItem& j) -> bool { return i.name < j.name; });
 
     return games;
 }
@@ -584,7 +581,7 @@ std::vector<wishlistItem> Website::getWishlistItems()
 
     do
     {
-        std::string response(this->getResponse("https://www.gog.com/account/wishlist/search?hasHiddenProducts=false&hiddenFlag=0&isUpdated=0&mediaType=0&sortBy=title&system=&page=" + std::to_string(i)));
+        std::string response(this->getResponse("https://www.gog.com/account/wishlist/search?hasHiddenProducts=false&hiddenFlag=0&isUpdated=0&mediaType=0&system=&page=" + std::to_string(i)));
         std::istringstream response_stream(response);
 
         try {
